base/js/admin.js

```diff
@@ -838,8 +838,13 @@ var sowbForms = window.sowbForms || {};
 							$inputElement.parent().empty().append($inputElement);
 							$inputElement.css('display', '');
 							var curEd = tinymce.get(id);
-							if (curEd) {
-								$inputElement.val(curEd.getContent());
+							if ( curEd ) {
+								var contentVal = curEd.getContent();
+								if ( ! _.isEmpty( contentVal ) ) {
+									$inputElement.val( contentVal );
+								} else if ( contentVal.search( '<' ) !== -1 && contentVal.search( '>' ) === -1) {
+									$textarea.val( contentVal.replace( /</g, '' ) );
```

Comment:
@AlexGStapleton what's this specific line for? Just for my own understanding. A comment might also help here. Is this some sort of tag stripping?

Reply:
This is actually a duplicate of the TinyMCE form field JS file. https://github.com/siteorigin/so-widgets-bundle/blob/develop/base/inc/fields/js/tinymce-field.js#L121

It searches the content for that has an opening < but not a closing >. In that situation, it strips <. This is done to prevent potentially malformed HTML. In hindsight, I'm not sure how useful either situation (both in this PR and in the initial TinyMCE field) as it could very well cause issues for those who use <.

Hm. Should I remove both instances?

Reply:
I can't actually offer any solid advice here, but I understand what this is for now. It works for me.

Reply:
Honestly, this has never come up so I think it's best to just leave it until we get a few reports as this could actually be resolving a weird edge case.